perma_web/frontend/pages/dashboard.js

```diff
@@ -1,3 +1,39 @@
 import { createApp } from 'vue'
 import App from '../components/App.vue'
+import { watch } from 'vue';
+import { globalStore } from '../stores/globalStore';
+
 createApp(App).mount('#vue-app')
+
+const dashboardTestDiv = document.getElementsByClassName("vanilla-div")[0]
```

Comment:
I don't know that we necessarily need to keep this in all one file — but including it here for now for the purpose of readability